reference/management/openapi.yaml

```diff
@@ -0,0 +1,1540 @@
4ec46a86f9bf71430c53cc590f9c07a6\"},\"/root/fullName\":{\"value\":\"Anna Gibbons\"},\"/payroll/employment/contract\":{\"value\":\"Full time\"},\"/payroll/employment/actualWorkingPattern\":{\"value\":{\"workingPatternType\":\"partial_week\",\"days\":{\"sunday\":false,\"tuesday\":true,\"wednesday\":true,\"monday\":true,\"friday\":true,\"thursday\":true,\"saturday\":false},\"hoursPerDay\":9,\"workingPatternId\":null}},\"companyId\":632127,\"/root/coverImageUrl\":{\"value\":\"https://media-process.hibob.com/image/upload/v1/hibob/public-image/cover/default_cover_image.png?token=X19jbGRfdG9rZW5fXz1leHA9NDg0OTgzNjczOX5hY2w9KiUyZmhpYm9iJTJmcHVibGljLWltYWdlJTJmY292ZXIlMmZkZWZhdWx0X2NvdmVyX2ltYWdlLnBuZyp+aG1hYz0zMjY3NjE4ZDQ1MzZjZjE3MWFiM2Y2MzJlY2JmZmQ5MWZkYTRmY2FmOGY3MDExYjllNmZlZTI4Njc0MzZlN2M5&vendor=cloudinary\"},\"/work/secondLevelManager\":{\"value\":\"3145813528661721978\"},\"/work/yearsOfService\":{\"value\":8.542},\"/work/daysOfPreviousService\":{\"value\":0},\"/about/about\":{\"value\":\"I dont have a life story, my story is my life \"},\"/root/avatarUrl\":{\"value\":\"https://media-process.hibob.com/image/upload/v1/hibob/demo-avatar/demo_employee/images/UiRYuQQyDc4sLbYzFFtN_59.jpg?__cld_token__=exp=4795680381~acl=*%2fhibob%2fdemo-avatar%2fdemo_employee%2fimages%2fUiRYuQQyDc4sLbYzFFtN_59.jpg*~hmac=07e5aff698027029ebbb4d8cb067db044ec46a86f9bf71430c53cc590f9c07a6\"},\"/root/surname\":{\"value\":\"Gibbons\"},\"/work/shortStartDate\":{\"value\":\"03-03\"},\"/work/activeEffectiveDate\":{\"value\":\"2015-03-03\"},\"/root/id\":{\"value\":\"3145813590838084494\"},\"email\":\"anna.gibbons_632127@samplebob.com\",\"/work/manager\":{\"value\":\"3145813558047015811\"},\"surname\":\"Gibbons\",\"/work/reportsTo\":{\"value\":{\"displayName\":\"Joshua Tyler\",\"email\":\"joshua.tyler_632127@samplebob.com\",\"surname\":\"Tyler\",\"firstName\":\"Joshua\",\"id\":\"3145813558047015811\"}},\"/root/displayName\":{\"value\":\"Anna Gibbons\"},\"/root/companyId\":{\"value\":632127},\"/payroll/employment/fte\":{\"value\":100},\"/work/department\":{\"value\":\"Sales\"},\"/work/isManager\":{\"value\":false},\"/work/custom/field_1693264337015\":{\"value\":580192},\"/work/reportsToIdInCompany\":{\"value\":27},\"/work/tenureDuration\":{\"value\":{\"periodISO\":\"P8Y6M15D\",\"sortFactor\":3075,\"humanize\":\"8 years, 6 months and 15 days\"}},\"coverImageUrl\":\"https://media-process.hibob.com/image/upload/v1/hibob/public-image/cover/default_cover_image.png?token=X19jbGRfdG9rZW5fXz1leHA9NDg0OTgzNjczOX5hY2w9KiUyZmhpYm9iJTJmcHVibGljLWltYWdlJTJmY292ZXIlMmZkZWZhdWx0X2NvdmVyX2ltYWdlLnBuZyp+aG1hYz0zMjY3NjE4ZDQ1MzZjZjE3MWFiM2Y2MzJlY2JmZmQ5MWZkYTRmY2FmOGY3MDExYjllNmZlZTI4Njc0MzZlN2M5&vendor=cloudinary\",\"/about/superpowers\":{\"value\":[\"proof reading\"]},\"/work/siteId\":{\"value\":2477371},\"id\":\"3145813590838084494\",\"firstName\":\"Anna\",\"/work/durationOfEmployment\":{\"value\":{\"periodISO\":\"P8Y6M15D\",\"sortFactor\":3075,\"humanize\":\"8 years, 6 months and 15 days\"}}}]}'''
```

Comment:
this renders a bit weird (multiple slashes like in your screenshot). is that intended?

Reply:
Not intended. Updated. Thank you. 🙇‍♂️